include/aikido/planner/dart/ConfigurationToEndEffectorPose.hpp

```diff
@@ -14,19 +14,38 @@ namespace dart {
 class ConfigurationToEndEffectorPose : public Problem
 {
 public:
-  /// Constructor.
+  /// Constructor. Note that this constructor takes the start state from the
+  /// current state of the passed MetaSkeleton.
   ///
   /// \param[in] stateSpace State space.
+  /// param[in] metaSkeleton MetaSkeleton that getStartState will return the
+  /// current state of when called.
   /// \param[in] endEffectorBodyNode BodyNode to be planned to move to a desired
   /// pose.
-  /// \param[in] startState Start state.
   /// \param[in] goalPose Goal pose.
   /// \throw If \c stateSpace is not compatible with \c constraint's state
   /// space.
   ConfigurationToEndEffectorPose(
       statespace::dart::ConstMetaSkeletonStateSpacePtr stateSpace,
+      ::dart::dynamics::ConstMetaSkeletonPtr metaSkeleton,
       ::dart::dynamics::ConstBodyNodePtr endEffectorBodyNode,
+      const Eigen::Isometry3d& goalPose,
+      constraint::ConstTestablePtr constraint);
```

Comment:
Nit: Let's pass `nullptr` to constraint as the default value for the same reason.